lib/playback/index.ts

```diff
@@ -135,6 +138,29 @@ export class Player {
 		}
 	}
 
+	displayCatalog() {
```

Comment:
This should be done in `watch.tsx` instead. It already has access to the catalog (it's passed in the constructor).

Also you should use SolidJS (similar to React) instead of VanillaJS to render/hide it. I can help if you're completely unfamiliar and uncomfortable.

Reply:
Thank you for the review, replaced displayCatalog with getCatalog to make the catalog-json accessible/visible to `watch.tsx`, and shifted the display/download part to `watch.tsx`. 
I am new to SolidJS, but eager to learn, and have attempted to implement the render/hide using SolidJS, please review and advise any amendments to be done